packages/core/strapi/lib/commands/__tests__/admin-reset.test.js

```diff
@@ -30,7 +30,7 @@ describe('admin:reset-password command', () => {
 
   test('resetAdminPasswordCommand accepts direct input', async () => {
     const email = 'email@email.fr';
-    const password = 'testPasword1234';
+    const password = 'testPassword1234';
```

Comment:
I understand this is garbage, but I don't see any reason it needs to be misspelled